normalizer/src/main/scala/tech/sourced/berserker/normalizer/model/Schema.scala

```diff
@@ -0,0 +1,14 @@
+package tech.sourced.berserker.normalizer.model
+
+import org.apache.spark.sql.types.{StringType, StructField, StructType}
+
+object Schema {
+  val files = StructType(
+    StructField("repoId", StringType, nullable = false) ::
+      StructField("hash", StringType, nullable = false) ::
+      StructField("path", StringType, nullable = false) ::
+      StructField("lang", StringType, nullable = true) ::
+      StructField("ast", StringType, nullable = true) ::
```

Comment:
Could you please double-check and make sure that all the [required fields](https://github.com/src-d/minutes/blob/master/engineering/2017-06-12-project-matching-demo-data-science-data-retrieval.md#notes) are in the schema?

What do you think about ast -> uast, to set the expectations for consumers?